docs/release/fundamentals/tables-and-data-operations.ipynb

```diff
@@ -0,0 +1,2565 @@
+       "      <th>longitude</th>\n",
+       "      <th>latitude</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <td>0</td>\n",
+       "      <td>1.15</td>\n",
+       "      <td>10 km NW of Belfair, Washington</td>\n",
+       "      <td>2023-01-01 08:10:37.050</td>\n",
+       "      <td>-122.93</td>\n",
+       "      <td>47.51</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>1</td>\n",
+       "      <td>0.29</td>\n",
+       "      <td>23 km ENE of Ashford, Washington</td>\n",
+       "      <td>2023-01-02 01:02:43.950</td>\n",
+       "      <td>-121.76</td>\n",
+       "      <td>46.85</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>2</td>\n",
+       "      <td>0.2</td>\n",
+       "      <td>23 km ENE of Ashford, Washington</td>\n",
+       "      <td>2023-01-02 12:05:01.420</td>\n",
+       "      <td>-121.75</td>\n",
+       "      <td>46.86</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>3</td>\n",
+       "      <td>0.52</td>\n",
+       "      <td>15 km NNE of Ashford, Washington</td>\n",
+       "      <td>2023-01-02 12:45:14.220</td>\n",
+       "      <td>-121.95</td>\n",
+       "      <td>46.89</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>4</td>\n",
+       "      <td>1.56</td>\n",
+       "      <td>0 km WSW of Esperance, Washington</td>\n",
+       "      <td>2023-01-02 13:19:27.200</td>\n",
+       "      <td>-122.36</td>\n",
+       "      <td>47.79</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "   id  magnitude                           location               timestamp  \\\n",
+       "0   0       1.15    10 km NW of Belfair, Washington 2023-01-01 08:10:37.050   \n",
+       "1   1       0.29   23 km ENE of Ashford, Washington 2023-01-02 01:02:43.950   \n",
+       "2   2       0.20   23 km ENE of Ashford, Washington 2023-01-02 12:05:01.420   \n",
+       "3   3       0.52   15 km NNE of Ashford, Washington 2023-01-02 12:45:14.220   \n",
+       "4   4       1.56  0 km WSW of Esperance, Washington 2023-01-02 13:19:27.200   \n",
+       "\n",
+       "   longitude  latitude  \n",
+       "0    -122.93     47.51  \n",
+       "1    -121.76     46.85  \n",
+       "2    -121.75     46.86  \n",
+       "3    -121.95     46.89  \n",
+       "4    -122.36     47.79  "
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "eq_t.limit(5).collect()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `head()` and `tail()` methods provide an alternative way to do this. Pixeltable keeps track of the insertion order of all its data, and `head()` and `tail()` will always return the _earliest inserted_ and _most recently inserted_ rows in a table, respectively."
```

Comment:
head and tail are actually not alternatives to limit; maybe "provide a different way of doing this"?